docs/additional-features/reports.md

```diff
@@ -85,6 +85,9 @@ As you can see, reports are completely customizable. Validation logic can be as
 !!! warning
     Reports should never alter data: If you find yourself using the `create()`, `save()`, `update()`, or `delete()` methods on objects within reports, stop and re-evaluate what you're trying to accomplish. Note that there are no safeguards against the accidental alteration or destruction of data.
 
+!!! note 
```

Comment:
We don't need call-outs specifically for this; just a note within the text mentioning Markdown support is sufficient.

Reply:
Removed in commit 10c9954